### ISSUE TITLE
[Bug] unintended behavior for manifold-prop annotation parameters

### ISSUE THREAD

**Issue by CC007** [manifold-systems/manifold#483]
**Describe the bug**
When using `@var`, `@val`, `@get` and `@set` without parameter the resulting getter and/or setter is public by default. when you add `PropOption.Private` or `PropOption.Package` as an annotation parameter, the created method for the respective getter/setter will be private or package-private respectively. So far so good.

However for `@get`, `@set` and `@var`, when using `PropOption.Public` or `PropOption.Protected`, I get the following error:
```
java: 'get' attempting to assign weaker access privileges; was 'package'
```
If manifold were to follow the java standard and make the getters and setters package-private by default, I would understand this error, but since manifold makes these public by default, this error seems incorrect.

Also, for `@val` I get a different error: 
```
java: java.lang.NullPointerException: Cannot invoke "com.sun.tools.javac.tree.JCTree.getStartPosition()" because "tree" is null
java: java.lang.NullPointerException: Cannot invoke "com.sun.tools.javac.tree.JCTree.getStartPosition()" because "tree" is null
```

**Workaround**
Marking the field as public solves the issue for all 4 cases and still properly makes the field private after compilation, but this seems redundant, since this should have been the default with Manifold.

**To Reproduce**
1. Build code from gist/picture

**Expected behavior**
I expected the use of Public and Protected to work as well, with all 4 annotations.

**Screenshots**
![afbeelding](https://github.com/manifold-systems/manifold/assets/5381337/bb03d7ff-3acd-4eee-868d-0499929bb858)

**Desktop (please complete the following information):**
- OS Type & Version: Windows 10
- Java/JDK version: openjdk-20.0.2
- IDE version (IntelliJ IDEA or Android Studio): IntelliJ 2023.1.5
- Manifold version: 2023.1.18
- Manifold IntelliJ plugin version: 23.1.15

**Additional context**
Code: https://gist.github.com/CC007/afde6311db82c4f57cf8b0496f34c471

*PS: sorry for posting so many issues in one day* 😅

**Comment by rsmckinney** [comment]
Don't apologize for reporting bugs. It is the best way to improve the project!  Thank you

**Comment by rsmckinney** [comment]
Fix available with release 2023.1.19. Thanks for reporting this!

**Comment by CC007** [comment]
Fix confirmed
